src/api/services/charactersService.ts

```diff
@@ -45,4 +54,20 @@ export namespace CharactersService {
     );
     return PaginationMapper.mapPaginationFromDto(data, CharacterMapper.fromDto).items[0];
   };
+
+  /**
+   * Gets a page of characters.
+   */
+  export async function fetchPaginatedCharacters(options: CharacterSearchOptions): Promise<Pagination<Character>> {
+    const characterOptions = [PaginationMapper.mapOptionsToDto(options)];
+
+    const characterFilters = [`search ~ "${options.search ?? ''}"`];
```

Comment:
We need to think about whether we can make a reusable function for this parsing step (search params to vndb search query) or not 🤔

Reply:
I think it's ok for now